### DISCUSSION TITLE
Sn5 Modified Planner 2024-27

### DISCUSSION THREAD

**Discussion by tchoffman** [kudrykv/latex-yearly-planner#102] [Show and tell]
These are MOS (months on side) planners for Supernote 5.  
1) I modified the "Week" page to have much smaller days and a larger "Weekly Goals" section.  Saturday/Sunday are combined into a non-hyperlinked "Weekend".  
2) Modified the Daily Reflection Page to have a single small section called "Remember When:".  I record cute things my kids say, big wins at work, impactful world events and such.  Basically, it's fun to skim and "remember when" these things happened.  I then made a bigger section called Daily goals.  I usually draw a grid here where I record on the left side what I ate for Breakfast, Lunch, Dinner and Snacks.  Then on the right I have a few checkboxes for "Avoid Sugar, Avoid Alcohol, Bedtime 10pm, Exercise 30 min, Reading 30 min".  I just copy this with Supernote Lasso tool onto each day as my goals might change during the year.

Anyway, after a year using the planner I realized how I use it and made some modifications.

I'm a software engineer myself... Maybe in 15 years when my kids are all moved out I'll open a few PRs here to add some customization.  😉   Would be awesome if there were a UI to configure each page.
[sn_a5x.mos.default custom 2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/13780678/sn_a5x.mos.default.custom.2024.pdf)
[sn_a5x.mos.default custom 2025.pdf](https://github.com/kudrykv/latex-yearly-planner/files/13780679/sn_a5x.mos.default.custom.2025.pdf)
[sn_a5x.mos.default custom 2026.pdf](https://github.com/kudrykv/latex-yearly-planner/files/13780680/sn_a5x.mos.default.custom.2026.pdf)
[sn_a5x.mos.default custom 2027.pdf](https://github.com/kudrykv/latex-yearly-planner/files/13780681/sn_a5x.mos.default.custom.2027.pdf)

**Comment by Cockpitinferno** [comment]
really great, are you able to do otherthings, could you check my discussion here please? 
https://github.com/kudrykv/latex-yearly-planner/discussions/100

if you could do something like this or explain how to do?
thanks a lot.
